jmxtrans-core/pom.xml

```diff
@@ -82,6 +82,10 @@
 			<artifactId>commons-cli</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>commons-configuration</groupId>
```

Comment:
I'm trying to reduce the number of dependencies of jmxtrans-core. If you could find a way to do some inversion of control here and extract as much as possible of the clustering code to a separate module, that would be great!